aaelf64-morello/aaelf64-morello.rst

```diff
@@ -0,0 +1,831 @@
+
+- The symbol is not a mapping symbol (`Mapping symbols`_).
+
+- The resulting object, or image, is not required to preserve accurate symbol
+  information to permit de-compilation or other post-linking optimization
+  techniques.
+
+- If the symbol labels an object in a section with the ``SHF_MERGE`` flag set,
+  the relocation using symbol may be changed to use the section symbol only if
+  the initial addend of the relocation is zero.
+
+No tool is required to perform the above transformations; an object consumer
+must be prepared to do this itself if it might find the additional symbols
+confusing.
+
+Mapping symbols
+^^^^^^^^^^^^^^^
+
+A section of an ELF file can contain a mixture of A64 code, C64 code and data.
+There are inline transitions between code and data at literal pool boundaries.
+
+Linkers, file decoders and other tools need to map binaries correctly. To
+support this, a number of symbols, termed mapping symbols appear in the symbol
+table to label the start of each sequence of bytes of the appropriate class. All
+mapping symbols have type ``STT_NOTYPE`` and binding ``STB_LOCAL``. The
+``st_size`` field is unused and must be zero.
+
+The mapping symbols are defined in `Mapping symbols table`_. It is an error for
+a relocation to reference a mapping symbol. Two forms of mapping symbol are
+supported:
+
+- A short form that uses a dollar character and a single letter denoting the
+  class. This form can be used when an object producer creates mapping symbols
+  automatically. Its use minimizes string table size.
+
+- A longer form in which the short form is extended with a period and then any
+  sequence of characters that are legal for a symbol. This form can be used when
+  assembler files have to be annotated manually and the assembler does not
+  support multiple definitions of symbols.
+
+Mapping symbols defined in a section (relocatable view) or segment (executable
+view) define a sequence of half-open intervals that cover the address range of
+the section or segment. Each interval starts at the address defined by the
+mapping symbol, and continues up to, but not including, the address defined by
+the next (in address order) mapping symbol or the end of the section or segment.
+A section that contains instructions must have a mapping symbol defined at the
+beginning of the section. If a section contains only data no mapping symbol is
+required. A platform ABI should specify whether or not mapping symbols are
+present in the executable view; they will never  be present in a stripped
+executable file.
+
+.. _Mapping symbols table:
+
+.. class:: aaelf64-morello-mapping-symbols
+
+.. table:: Mapping symbols
+
+    +----------------+------------------------------------------------------------------+
+    | Name           | Meaning                                                          |
+    +================+==================================================================+
```

Comment:
Please end sentences starting with a Capital with a period`.`

Reply:
Done. But does not show as `outdated` in the UI for some reason.